### ISSUE TITLE
Reusable Model

### ISSUE THREAD

**Issue by shantanutyagi67** [arkaprabha10/computer_vision#1]
It is a good practice to save the ML you just trained so that you can use it in the future.
`model.save("fashion.model")`
load the model in any file to make predictions
`model = tf.keras.models.load_model('fashion.model')`
`result = model.predict(test_image)`
This will give softmax node values, from which our answer is
`np.argmax(result)`

**Comment by arkaprabha10** [comment]
That's certainly helpful! I shall definitely incorporate it in my future codes.
